insonmnia/dwh/server.go

```diff
@@ -391,9 +391,8 @@ func (m *DWH) watchMarketEvents() error {
 			}
 			m.processBlockBoundary(event)
 			dispatcher.Add(event)
-			if eventsCount < m.cfg.NumWorkers {
-				eventsCount++
-			} else {
+			eventsCount++
```

Comment:
With this change exactly `m.cfg.NumWorkers` events can be processed in parallel (not `m.cfg.NumWorkers + 1`, as in the old version).